package.json

```diff
@@ -86,16 +87,19 @@
   "devDependencies": {
     "@commitlint/cli": "^17.1.2",
     "@commitlint/config-conventional": "^17.1.0",
+    "@krakenjs/grabthar-release": "^3.8.0",
```

Comment:
```suggestion
```

Can we remove grabthar-release? I don't think we need it and I don't see it used anywhere

Reply:
Weird, I thought I just did that...either way good catch!